pages/docs/adapters/calculix/adapter-calculix-configure.md

```diff
@@ -23,30 +23,37 @@ precice-config-file: ../precice-config.xml
 ```
 
 The adapter allows to use several participants in one simulation (e.g. several instances of Calculix if several solid objects are taken into account). The name of the participant "Calculix" must match the specification of the participant on the command line when running the executable of "CCX" with the adapter being used (this is described later). Also, the name must be the same as the one used in the preCICE configuration file *precice-config.xml*.  
-One participant may have several FSI interfaces. Note that each interface specification starts with a dash.  
-For FSI simulations the mesh type of an interface is always "nodes-mesh", i.e. the mesh is defined node-wise, not element-wise. The name of this mesh, "Calculix_Mesh", must match the mesh name given in the preCICE configuration file.  
+One participant may have several coupling interfaces. Note that each interface specification starts with a dash.
+Depending on the data you need to read and write, you can use a mesh made of face centers ("faces-mesh" or just "mesh") or a mesh made of actual CalculiX nodes.
+For FSI simulations the mesh type of an interface is always "nodes-mesh", as forces and displacement are defined on nodes. The name of this mesh, "Calculix_Mesh", must match the mesh name given in the preCICE configuration file. In CHT simulations, faces mesh are usually chosen, as they are needed to apply heat fluxes or convective heat transfer.
 For defining which nodes of the CalculiX domain belong to the FSI interface, a node set needs to be defined in the CalculiX input files. The name of this node set must match the name of the patch (here: "interface").  
 In the current FSI example, the adapter reads forces from preCICE and feeds displacement deltas (not absolute displacements, but the change of the displacements relative to the last time step) to preCICE. This is defined with the keywords "read-data" and "write-data", respectively. The names (here: "Forces" and "DisplacementDeltas") again need to match the specifications in the preCICE configuration file. In the current example, the coupled fluid solver expects displacement deltas instead of displacements. However, the adapter is capable of writing either type. Just use "write-data: [Displacements]" for absolute displacements rather than relative changes being transferred in each time step. Valid `readData` keywords in CalculiX are:
 
 ```text
-* Forces
-* Displacements
-* Temperature
-* Heat-Flux
-* Sink-Temperature
-* Heat-Transfer-Coefficient
+On faces-mesh:
+* Pressure (Apply a *DLOAD)
+* Heat-Flux (Apply a *DFLUX)
+* Sink-Temperature (Use *FILM)
+* Heat-Transfer-Coefficient (Use *FILM)
+On nodes-mesh:
+* Forces (Apply a *CLOAD)
+* Displacements (Use *BOUNDARY)
+* Temperature (Use *BOUNDARY)
+
 ```
```

Comment:
Why do we actually have this in a code block? Better convert to two lists with inline code where needed (same below).